tests/network-tests/src/flows/proposals/channelPayouts.ts

```diff
@@ -93,15 +93,15 @@ export default async function channelPayouts({ api, query, lock, env }: FlowProp
   // Upload channel payouts payload to the council bag
   await joystreamCli.reuploadAssets({
     bagId: COUNCIL_BAG_ID,
-    assets: [{ objectId: channelPayoutsUpdatedEvent.payloadDataObject.id, path: protobufPayloadFilePath }],
+    assets: [{ objectId: channelPayoutsUpdatedEvent.payloadDataObject?.id || '', path: protobufPayloadFilePath }],
```

Comment:
If the payload data object is expected to never be `null`/`underfined` here, it's better to use:
```typescript
Utils.assert(channelPayoutsUpdatedEvent.payloadDataObject, 'Payload data object missing!')
```